cpp/src/copying/contiguous_split.cu

```diff
@@ -821,7 +821,8 @@ void copy_data(int num_bufs,
     chunks.begin(),
     [desired_chunk_size] __device__(dst_buf_info const& buf) -> thrust::pair<size_t, size_t> {
       // Total bytes for this incoming partition
-      size_t const bytes = buf.num_elements * buf.element_size;
+      size_t const bytes =
+        static_cast<size_t>(buf.num_elements) * static_cast<size_t>(buf.element_size);
```

Comment:
```suggestion
      auto const bytes =
        static_cast<std::size_t>(buf.num_elements) * static_cast<std::size_t>(buf.element_size);
```
Could you include `<cstddef>` and replace all `::size_t` with `std::size_t` throughout this function? The rest of the file is pretty consistent of using `std::size_t` instead of `::size_t`.

Reply:
Thanks for the review!  Yes I will make this change.